packages/pipeline/test/entities/ohlcv_external_test.ts

```diff
@@ -0,0 +1,35 @@
+import 'mocha';
+import 'reflect-metadata';
+
+import { OHLCVExternal } from '../../src/entities';
+import { createDbConnectionOnceAsync } from '../db_setup';
+import { chaiSetup } from '../utils/chai_setup';
+
+import { testSaveAndFindEntityAsync } from './util';
+
+chaiSetup.configure();
+
+const ohlcvExternal = {
```

Comment:
Small tip that I just recently discovered, if you add the `OHLCVExternal` type to `ohlcvExternal`, the compiler will alert you if you are missing any fields or have any types that are incorrect 😄 . To take advantage of this you would just write:


```ts
const ohlcvExternal: OHLCVExternal = {
    // ...
}
```